legend-pure-code-compiled-core-relational/src/main/resources/core_relational/relational/sqlQueryToString/SQLQueryToString.pure

```diff
@@ -1,2539 +1,2545 @@
-   let dateTimeFormat = if( $params->size() == 1,|'\'YYYY-MM-DD HH24:MI:SS\'',| $params->at(1));
-   //https://docs.snowflake.net/manuals/sql-reference/data-types-datetime.html
-   'to_timestamp('+$params->at(0)+','+$dateTimeFormat +')';
-}
-
-function <<access.private>> meta::relational::functions::sqlQueryToString::dateFormats():Map<String,Integer>[1]
-{
-   newMap([pair('yyyy-MM-dd',120),
-           pair('MMMyyyy', 106),
-           pair('dd MMM yy',6),
-           pair('yyyyMMdd', 112)]);
-}
-
-function <<access.private>> meta::relational::functions::sqlQueryToString::dateFormatsMemSQL():String[*]
-{
-   ['YYYY-MM-DD', 'MMMYYYY', 'YYYYMMDD'];
-}
-
-function <<access.private>> meta::relational::functions::sqlQueryToString::dateFormatsH2():String[*]
-{
-   ['yyyy-MM-dd', 'MMMyyyy', 'yyyyMMdd'];
-}
-
-function <<access.private>> meta::relational::functions::sqlQueryToString::dateFormatsDatabricks():String[*]
-{
-   ['yyyy-MM-dd', 'MMMyyyy', 'yyyyMMdd'];
-}
-
-function <<access.private>> meta::relational::functions::sqlQueryToString::dateFormatsSnowFlake():String[*]
-{
-   ['yyyy-MM-dd', 'MMMyyyy', 'yyyyMMdd','DD/MM/YYYY', 'YYYY/MM/DD'];
-}
-
-function <<access.private>> meta::relational::functions::sqlQueryToString::dateTimeFormats():Map<String,Integer>[1]
-{
-   newMap([pair('yyyy-mm-dd hh:mi:ss',120),
-           pair('yyyy-MM-dd hh:mm:ss',120),
-           pair('yyyy-MM-dd hh:mm:ss.mmm',121)]);
-}
-
-function <<access.private>> meta::relational::functions::sqlQueryToString::dateTimeFormatsSnowFlake():Map<String,Integer>[1]
-{
-   newMap([pair('yyyy-mm-dd hh:mi:ss',120),
-           pair('yyyy-MM-dd hh:mm:ss',120),
-           pair('yyyy-MM-dd hh:mm:ss.mmm',121),
-           pair('YYYY-MM-DDTHH:MI:SS',120),
-           pair('YYYY-MM-DDTHH:MI:SS.FF', 121)]);
-}
-
-function <<access.private>> meta::relational::functions::sqlQueryToString::transformPercentile(args: String[*], dbType: DatabaseType[1]): String[1]
-{
-   let boolProc = getLiteralProcessorForType(getLiteralProcessors($dbType, []), Boolean);
-   let trueVal  = $boolProc.formatValue(true);
-   let falseVal = $boolProc.formatValue(false);
-
-   let column     = $args->at(0);
-   let percentile = $args->at(1);
-   let ascending  = if($args->size() > 2, | $args->at(2), | $trueVal);
-   let continuous = if($args->size() > 3, | $args->at(3), | $trueVal);
-
-   assert($ascending->in([$trueVal, $falseVal]), 'Ascending argument for percentile should be ' + $trueVal + ' or ' + $falseVal + '. Got: ' + $ascending);
-   assert($continuous->in([$trueVal, $falseVal]), 'Continuous argument for percentile should be ' + $trueVal + ' or ' + $falseVal + '. Got: ' + $continuous);
-
-   let func  = if($continuous == $trueVal, | 'percentile_cont', | 'percentile_disc');
-   let order = if($ascending == $trueVal, | 'asc', | 'desc');
-
-   $func + '(' + $percentile + ') within group (order by ' + $column + ' ' + $order + ')';
-}
+// Copyright 2021 Goldman Sachs
```

Comment:
Per @vj2050gs comment, this file use to be on CRLF, but was changed to LF.  Given the majority of the files is LF, this seems correct.  To review, hide the whitespace.